docs/protocol/whitefox-protocol-api.yml

```diff
@@ -576,7 +576,7 @@ paths:
           $ref: "#/components/responses/409"
         "500":
           $ref: "#/components/responses/500"
-  /whitefox-api/v1/storage:
```

Comment:
[storage is an uncountable noun](https://www.oxfordlearnersdictionaries.com/definition/english/storage#:~:text=storage-,noun,where%20things%20can%20be%20kept) 😄